src/NzbDrone.Core/ImportLists/Rss/RssImportBase.cs

```diff
@@ -0,0 +1,45 @@
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Parser;
+
+namespace NzbDrone.Core.ImportLists.Rss
+{
+    public class RssImportBase<TSettings> : HttpImportListBase<TSettings>
+        where TSettings : RssImportBaseSettings, new()
+    {
+        public override string Name => "RSS List Base";
```

Comment:
If this is meant to be an abstract class, there is no need for a name or type here. They should be set at the inherited level